roxie/roxiemem/roxiemem.cpp

```diff
@@ -3415,8 +3415,8 @@ class BufferedRowCallbackManager
     Owned<BackgroundReleaseBufferThread> backgroundReleaseBuffersThread;
     Owned<ReleaseBufferThread> releaseBuffersThread;
     CRowManager * owner;
-    atomic_t releasingBuffers;  // boolean if pre-emptive releasing thread is active
-    atomic_t releaseSeq;
+    std::atomic_uint releasingBuffers;  // boolean if pre-emptive releasing thread is active
```

Comment:
The reason it is a uint instead of a boolean was that the power8 generated poor code for an atomic  boolean.

I agree about the consistency.  Which do you prefer?  Always prefixing with std:: or using a typedef so it isn't qualified?  I think I may prefer always qualifying, but no strong opinion.